app/views/foi/submissions/show.html.erb

```diff
@@ -53,10 +53,17 @@
       questions.
     </p>
 
+    <% if Performance.current_percentage %>
```

Comment:
I think we should aim to keep logic out of the view and assign this in the controller

```ruby
def show
  @current_performance = Performance.current_percentage
  # …
end
```

Its a bit pedantic, but makes it easier to test views by just assigning instance vars.